staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -204,7 +204,7 @@ func checkKeyFormat(key string) error {
 	if len(parts) != 2 {
 		return fmt.Errorf("annotation key has invalid format, the right format is a DNS subdomain prefix and '/' and key name. (e.g. 'podsecuritypolicy.admission.k8s.io/admit-policy')")
 	}
-	if msgs := validation.IsQualifiedName(key); len(msgs) != 0 {
+	if msgs := validation.IsQualifiedName(strings.ToLower(key)); len(msgs) != 0 {
```

Comment:
The validation in metadata.annotation is a little confusing today https://github.com/kubernetes/kubernetes/pull/109892. Not sure if we should change the validation in audit annotation to match that, or simply improve our documentation.

Reply:
ok, i will close it. 
tks